localization/english/religion/wc_religion_l_english.yml

```
@@ -192,6 +192,12 @@
  scarlet_light_adherent:0 "Scarlet Follower"
  scarlet_light_adherent_plural:0 "Scarlet Followers"
  scarlet_light_desc:0 "A heretical, militant offshoot of the Church of the Holy Light. They are intolerant of all other faiths, regardless of alliance or affiliation. They believe that any and all unholy creatures must be destroyed by a cleansing flame."
+
+ kirin_tor:0 "The Kirin Tor"
+ kirin_tor_adj:0 "The Kirin Tor"
+ kirin_tor_adherent:0 "Kirin Tor Mage"
+ kirin_tor_adherent_plural:0 "Kirin Tor Mages"
```

Inventing new words.
Mage isn't good because no every adherent is a mage.
```suggestion
 kirin_tor:0 "Kirin Torism"
 kirin_tor_adj:0 "Kirin Toran"
 kirin_tor_adherent:0 "Kirin Toran"
 kirin_tor_adherent_plural:0 "Kirin Torans"
```